ci/docker/python-wheel-manylinux-x-y.dockerfile

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARG base
+FROM ${base}
+
+ARG arch
+ARG arch_short
+ARG manylinux
+
+ENV MANYLINUX_VERSION=${manylinux}
+
+# Install basic dependencies
+RUN yum install -y git flex curl autoconf zip perl-IPC-Cmd wget
+
+ENV HOST_PYTHON_VERSION=3.8
+RUN HOST_PYTHON_ROOT=$(find /opt/python -name cp${HOST_PYTHON_VERSION/./}-*) && \
+    echo "export PATH=$HOST_PYTHON_ROOT/bin:\$PATH" >> /python.sh
+
+# Install CMake
+# AWS SDK doesn't work with CMake=3.22 due to https://gitlab.kitware.com/cmake/cmake/-/issues/22524
+ARG cmake=3.21.4
+COPY ci/scripts/install_cmake.sh arrow/ci/scripts/
+RUN source /python.sh && /arrow/ci/scripts/install_cmake.sh ${arch} linux ${cmake} /usr/local
+
+# Install Ninja
+ARG ninja=1.10.2
+COPY ci/scripts/install_ninja.sh arrow/ci/scripts/
+RUN source /python.sh && /arrow/ci/scripts/install_ninja.sh ${ninja} /usr/local
+
+# Install ccache
+ARG ccache=4.1
+COPY ci/scripts/install_ccache.sh arrow/ci/scripts/
+RUN source /python.sh && /arrow/ci/scripts/install_ccache.sh ${ccache} /usr/local
+
+RUN echo "Hello World"
+
+# Install vcpkg
+ARG vcpkg
+ARG glibc=2.28
```

Comment:
glibc set to 2.28

Reply:
Why? Could you add the reason as a comment of this line?

Reply:
Added PEP 600 comment

Reply:
Thanks.
But it seems that we can remove this and `ci/scripts/install_glibc.sh`. See also: #13566

Reply:
@sjperkins Could you check this?

Reply:
Yes, I will do so during the week.

Reply:
Removed `ci/scripts/install_glibc.sh`